data/ui/notes.ui

```diff
@@ -65,12 +67,13 @@
                     <property name="vexpand">true</property>
                     <property name="visible">false</property>
                     <property name="icon-name">document-edit-symbolic</property>
-                    <property name="title" translatable="true">No notes</property>
+                    <property name="title" translatable="true">No Saved Notes</property>
```

Comment:
not sure what you mean by "saved" notes, since all notes are "saved". this might make the user think there are saved and unsaved notes